src/registration/Registration.jsx

```diff
@@ -27,78 +33,159 @@ class Registration extends React.Component {
+                                        <Field
+                                            required={true} component={TextField} type="text"
+                                            name="address" label={t('register:address')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:address')}
+                                            className={classes.textField}
+                                        />
+                                    </Grid>
+                                    <Grid className={classes.field} item>
+                                        <Field
+                                            required={true} component={TextField} type="text"
+                                            name="postalCode" label={t('register:postalCode')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:postalCode')}
+                                            className={classes.textField}
+                                        />
+                                    </Grid>
+                                    <Typography>
+                                        Caring Calendar Administrator
+                                     </Typography>
+                                    <Grid className={classes.field} item>
+                                        <Field
+                                            required={true} component={TextField} type="text"
+                                            name="adminName" label={t('register:name')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:name')}
+                                            className={classes.textField}
+                                        />
+                                    </Grid>
+                                    <Grid className={classes.field} item>
+                                        <Field
+                                            required={true} component={TextField} type="text"
+                                            name="adminEmail" label={t('register:email')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:email')}
+                                            className={classes.textField}
+                                        />
+                                    </Grid>
+                                    <Grid className={classes.field} item>
+                                        <Field
```

Comment:
The password is being automatically generated once the organization has been verified.  The only information we are asking for for an admin user is first name, last name, and email